File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int n;
int space;

int main(void)
{
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 | n > 8);

    for (int i = 1; i <= n; i++)
    {
        space = n - i;
        for (int j = 0 ; j < space ; j++)
        {
            printf(" ");
        }

        for (int k = 0 ; k < i; k++)
        {
            printf("#");
        }

        printf("  ");

        for (int k = 0 ; k < i; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            piramid(n)
            break


def piramid(n, counter=0):
    hashtag = "#"
    space = " "

    if n == 0:
        return
    else:
        piramid(n-1, counter+1)
        text = space*counter + hashtag*n + space*2 + n*"#"
        print(f"{text}")


main()
